### DISCUSSION TITLE
textarea setting the text colour

### DISCUSSION THREAD

**Discussion by TomMann** [charmbracelet/bubbles#402] [Q&A] [answered]
Afternoon,
I was wondering if its possible to set the text colour of the textarea component,  I've tried setting it via a style but it doesn't seem to apply, it could be I'm holding it wrong (most likely) 

`style := lipgloss.NewStyle().Padding(1, 2).
				Border(lipgloss.RoundedBorder()).
				BorderForeground(lipgloss.Color("62")).
				Foreground(lipgloss.Color("9"))`
				
any assistance is appreciated :)

**Comment by naglis** [comment, marked as answer]
Yes, there is support to modify textarea styles in both [focused and blurred states](https://github.com/charmbracelet/bubbles/blob/95d7be54d9d4cdd73cd36e31e64b67965d517d68/textarea/textarea.go#L162-L163) and you can tweak how these different things should look in each state:
https://github.com/charmbracelet/bubbles/blob/95d7be54d9d4cdd73cd36e31e64b67965d517d68/textarea/textarea.go#L122-L129

Here is an snippet from the bubbletea examples on how to change the default styles:
https://github.com/charmbracelet/bubbletea/blob/91dd1200733714c4fb7bc7ffb24af2c35cc2f111/examples/split-editors/main.go#L55-L61

**Reply by TomMann** [reply]
Thanks, will give this a go, simply setting the style wasn't changing the font colour I see their are a lot more options on where you can assign styles so thanks for getting back to me 👍
